docs/editors-guide/strategic-refinement.md

```diff
@@ -2,17 +2,17 @@
 
 ## Summary
 
-The goal of this project is to to perform strategic refinement in Mondo and clean up the hierarchy. The first steps at this refinment is to remove the Orphanet (ORDO) grouping classes since these cause unwanted classifications in Mondo.
+The goal of this project is to perform strategic refinement in Mondo and refine the hierarchy. The first steps at this refinment is to remove the Orphanet (ORDO) grouping classes since these cause unwanted classifications in Mondo.
 
```

Comment:
Typo fixed here: 0d254488

Reply:
If there's a specific proposal I can update it, but if some discussion is needed let's use Slack :)

Reply:
The proposal is to change this sentence: 
"The first steps at this refinment is to remove the Orphanet (ORDO) grouping classes since these cause unwanted classifications in Mondo."

And replace it by: 
The first step at this refinement focuses on removing classes that group diseases by phenotypic features and "similar" diseases (eg:'dysostosis with predominant craniofacial involvement'- MONDO:0800085).

Reply:
updated 1cc72e6e